src/main/java/com/minecolonies/coremod/client/gui/GuiFactory.java

```diff
@@ -0,0 +1,36 @@
+package com.minecolonies.coremod.client.gui;
+
+import java.util.Set;
+
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.gui.GuiScreen;
+import net.minecraftforge.fml.client.IModGuiFactory;
+
+/**
+ * This class is required for Forge to initialize the in-game GUI.
+ */
+public class GuiFactory implements IModGuiFactory 
+{
+    @Override
+    public void initialize(Minecraft minecraftInstance) 
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1186)